### PR TITLE
Add Exercise 5

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_5/Program.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        static void ConvertJsonToXml(string jsonInputPath, string xmlOutputPath)
+        {
+            var xmlString = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(
+                Encoding.ASCII.GetBytes(ReadFromFile(jsonInputPath)), new XmlDictionaryReaderQuotas())).ToString();
+            WriteToFile(xmlString, xmlOutputPath);
+        }
+        static string ReadFromFile(string filePath)
+        {
+            string strOutput = "";
```
Comment (voquanghoa) [inline Exercise_5/Program.cs]: Return `File.ReadAllText(filePath)`

File: Exercise_5/Program.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+            return strOutput;
+        }
+
+        static void WriteToFile(string content, string filePath)
+        {
+            try
```
Comment (voquanghoa) [inline Exercise_5/Program.cs]: Use `File.WriteAllText(filePath, content)`

File: Exercise_5/Program.cs (hunk 1 above)
Comment (nguyenvanman) [reply Exercise_5/Program.cs]: Done

File: Exercise_5/Program.cs (hunk 2 above)
Comment (nguyenvanman) [reply Exercise_5/Program.cs]: Done

**Pull request merged**